app/src/typo.scss

```diff
@@ -16,3 +17,35 @@ $itero-mat-typography: mat-typography-config(
   $button:    mat-typography-level($fontsize, 1.2),
   $input:     mat-typography-level($fontsize, 1.3),
 );
+
+:root{
+  --font-size: 13pt;
+  --small-font-size: 12pt;
+}
+
+//laptop-size
+@media only screen and (min-width : $breakpoint-laptop)
+{
+  :root{
+        --font-size: 12pt;
+        --small-font-size: 11pt;
+       }
+}
```

Comment:
No. We don't want to change the default font size on large screen. It must remain 13pt.